gtsfm/utils/splat.py

```diff
@@ -263,3 +264,29 @@ def get_viewmat(camera_to_world: torch.Tensor) -> torch.Tensor:
     viewmat[:, :3, 3:4] = T_inv
     viewmat[:, 3, 3] = 1.0
     return viewmat
+
+
+def transform_gaussian(gaussianA, sim3_B_A):
+    """
+    Transforms a Gaussian Splat from one coordinate system to another using gtsam.Similarity3
+    Args:
+        gaussianA (dict): A dictionary representing the Gaussian in coordinate system A.
+        sim3_B_A (gtsam.Similarity3): The transformation from coordinate system A to B.
+    Returns:
+        gaussianB: A dictionary representing the Gaussian in coordinate system B.
+    """
+    meanA = gaussianA["means"]
+    meanB = sim3_B_A.transformFrom(meanA)
+
+    rotationBA = Rotation.from_matrix(sim3_B_A.rotation().matrix())
+    rotationB = (rotationBA * Rotation.from_quat(gaussianA["quats"])).as_quat()
```

Comment:
I guess this might be right but, naming convention says `bRa`. I’m a bit surprised gaussians are stored with just rotations and a scale and not the full covariance.. are there no “flattish” Gaussians?